### DISCUSSION TITLE
Case idea needed for project using Raspaudio+

### DISCUSSION THREAD

**Discussion by akivaatwood** [probonopd/MiniDexed#548] [Q&A]
Using the IO expansion board the audio board sticks out 8mm from the PI -- so either the usb sockets have to be inside the case or the audio board has to stick out

Is there a third option (short of running cables from all the sockets to a front panel)

TIA

**Comment by akivaatwood** [comment]
Decided to use jumpers instead of the header socket...so got rid of the IO board.

**Reply by Banana71** [reply]
It is often difficult to place the components correctly.  Please show us your miniDexed when you are finished. 🎉
